pkg/tests/zip_test.py

```diff
@@ -12,18 +12,27 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import datetime
 import filecmp
 import unittest
 import zipfile
 
 from bazel_tools.tools.python.runfiles import runfiles
-from rules_pkg.private import build_zip
 
 HELLO_CRC = 2069210904
 LOREM_CRC = 2178844372
 EXECUTABLE_CRC = 342626072
 
 
+# The ZIP epoch date: (1980, 1, 1, 0, 0, 0)
+_ZIP_EPOCH_DT = datetime.datetime(1980, 1, 1, 0, 0, 0, tzinfo=datetime.timezone.utc)
+_ZIP_EPOCH_S = int(_ZIP_EPOCH_DT.timestamp())
+
+def seconds_to_ziptime(s):
+  dt = datetime.datetime.utcfromtimestamp(s)
```

Comment:
Python's upstream recommendation is to use something like:

```python
datetime.datetime.fromtimestamp(s, datetime.timezone.utc)
```

Mostly b/c leap seconds.  This is probably fine for this use case, though.

[Documentation](https://docs.python.org/3.5/library/datetime.html#datetime.datetime.utcfromtimestamp).

Reply:
I'm inclined to keep this as is because it is the same code used in the build_zip tool.